.editorconfig

```diff
@@ -0,0 +1,77 @@
+csharp_new_line_before_open_brace = none:suggestion
+csharp_new_line_before_else = false:suggestion
+csharp_new_line_before_catch = false:suggestion
+csharp_new_line_before_finally = false:suggestion
+csharp_new_line_before_members_in_object_initializers = false:suggestion
+csharp_new_line_before_members_in_anonymous_types = false:suggestion
+csharp_new_line_between_query_expression_clauses = false:suggestion
+
+csharp_style_var_for_built_in_types = false:suggestion
+csharp_style_var_when_type_is_apparent = false:suggestion
+csharp_style_var_elsewhere = false:suggestion
+
+csharp_style_expression_bodied_methods = when_on_single_line:suggestion
+csharp_style_expression_bodied_constructors = when_on_single_line:suggestion
+csharp_style_expression_bodied_operators = when_on_single_line:suggestion
+csharp_style_expression_bodied_properties = when_on_single_line:suggestion
+csharp_style_expression_bodied_indexers = when_on_single_line:suggestion
+csharp_style_expression_bodied_accessors = when_on_single_line:suggestion
+
+csharp_style_pattern_matching_over_as_with_null_check = false:suggestion
+
+csharp_preferred_modifier_order = public,private,protected,internal,static,extern,new,virtual,abstract,sealed,override,readonly,unsafe,volatile,async:suggestion
+
+csharp_preserve_single_line_statements = false:suggestion
+csharp_preserve_single_line_blocks = true:suggestion
+
+csharp_indent_case_contents = true:suggestion
+csharp_indent_switch_labels = true:suggestion
+
+csharp_space_after_cast = true:suggestion
+csharp_space_after_keywords_in_control_flow_statements = true:suggestion
+csharp_space_between_method_declaration_parameter_list_parentheses = false:suggestion
+csharp_space_between_method_call_parameter_list_parentheses = false:suggestion
+csharp_space_between_parentheses = false:suggestion
+csharp_space_before_colon_in_inheritance_clause = true:suggestion
+csharp_space_after_colon_in_inheritance_clause = true:suggestion
+csharp_space_around_binary_operators = before_and_after:suggestion
+csharp_space_between_method_declaration_empty_parameter_list_parentheses = false:suggestion
+csharp_space_between_method_call_name_and_opening_parenthesis = false:suggestion
+csharp_space_between_method_call_empty_parameter_list_parentheses = false:suggestion
+
+
+dotnet_sort_system_directives_first = true:suggestion
+
+dotnet_style_require_accessibility_modifiers = always:suggestion
+dotnet_style_qualification_for_field = false:suggestion
+dotnet_style_qualification_for_property = false:suggestion
+dotnet_style_qualification_for_method = false:suggestion
+dotnet_style_qualification_for_event = false:suggestion
+dotnet_style_prefer_auto_properties = true:suggestion
+dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
+dotnet_style_predefined_type_for_member_access = true:suggestion
+
+# PascalCase for naming everything but parameters
+dotnet_naming_rule.public_members_must_be_capitalized.symbols = all_symbols
```

Comment:
Rename `public_members_must_be_capitalized` into `all_members_must_be_capitalized`, since it's more appropriate.

Reply:
done